### ISSUE TITLE
TypeScript support

### ISSUE THREAD

**Issue by tommmyy** [tommmyy/ramda-extension#86]
Add TS dictionary to @types

**Comment by Maus3rVonDutch** [comment]
Hi, this was ticket is opened over a year ago, is this still actively worked on, or is this issue abandoned altogether?

**Comment by tommmyy** [comment]
@lukasduspiva ?

**Comment by lukasduspiva** [comment]
Hi, the work on this task is postponed because we realized that functional JavaScript (which is the core of ramda-extension) is not well supported in strict TypeScript and we had several [problems](https://github.com/DefinitelyTyped/DefinitelyTyped/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+ramda) of using it even with Ramda functions which are already typed. Therefore it is not a big priority for us to write the typings. On the other hand, any volunteers with a focus on this domain are welcomed.

**Comment by Maus3rVonDutch** [comment]
Ok a bit disappointing, but I do understand. For now I only need a handful of extensions, so I think I can get way with writing some custom *.d.ts files. 

Thnx for the response and your explanation.

**Comment by tommmyy** [comment]
@Maus3rVonDutch you are welcome to help!

**Comment by Maus3rVonDutch** [comment]
Yes I understand I could, but I’m not able to do so. I no longer do any work at home anymore since a burnout a couple of years ago and in my work I don’t have the time to participate in any opensource projects.
